reach

```diff
@@ -1,4 +1,6 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Why did you need to turn it into bash? Normally shellcheck will tell you how to do the thing you want with sh